src/index.js

```diff
@@ -52,6 +53,12 @@ for (const file of eventFiles) {
   }
 }
 
+client.on('interactionCreate', async (interaction) => {
+  if (interaction.isModalSubmit()) {
+    await handleModalSubmit(interaction);
+    return;
+  }
+});
```

Comment:
This can be moved to the folder /events, it will be run automatically from there, so you dont have to create the event manually here. 

Then you can remove it from `index.js`